lib/commands/sync.js

```diff
@@ -54,9 +54,31 @@ let HELPTEXT = `
     --indexesOnly                             Just create indexes
     --buffer                                  <number> - Minimum number of connections available in the pool, default 20
     --max                                     <number> - Maximum number of connections available in the pool, default 50
+    --allMessages                             Pulls all messages instead of last 3 months (default: false)
```

Comment:
In the future we can add the ability to pass in the range of messages to sync.... B U T  O F   C O U R S E  T H A T ' S    N O T   I M P O R T A N T   N O W